src/Honeybee_ Run Energy Simulation.py

```diff
@@ -26,8 +26,9 @@
             8 - City terrian.
         +++++++++++++++: ...
         _HBZones: The HBZones that you wish to write into an IDF and/or run through EnergyPlus.  These can be from any of the components that output HBZones.
-        HBContext_: Optional HBContext geometry from the "Honeybee_EP Context Surfaces." component.
-        simulationOutputs_: A list of the outputs that you would like EnergyPlus to write into the result CSV file.  This can be any set of any outputs that you would like from EnergyPlus, writen as a list of text that will be written into the IDF.  It is recommended that, if you are not expereinced with writing EnergyPlus outputs, you should use the "Honeybee_Write EP Result Parameters" component to request certain types of common outputs.  If no value is input here, this component will automatically request outputs of heating, cooling, lighting, and equipment energy use.  
+        HBContext_: Optional HBContext geometry from the "Honeybee_EP Context Surfaces." component or Honeybee PV gen component.
+        HB_generators: Connect the output HB_generatorsytem from the Honeybee_generationsystem component here to model EnergyPlus Photovoltaic and Wind generator systems in this simulation.
```

Comment:
@antonszilasi HB_generators should be HBGenerators_
Just as a reminder, here is the naming convention:
_requiredInput
_inputsWithDefaultValue_
optionalInput_